Add the linux,spidev compatible in spidev
Several device in ramips have this binding in the dts

Signed-off-by: Giuseppe Lippolis <giu.lippolis@gmail.com>
---
--- a/drivers/spi/spidev.c
+++ b/drivers/spi/spidev.c
@@ -696,6 +696,7 @@ static struct class *spidev_class;
 	{ .compatible = "lineartechnology,ltc2488" },
	{ .compatible = "ge,achc" },
 	{ .compatible = "semtech,sx1301" },
+	{ .compatible = "siliconlabs,si3210" },
 	{},
 };
 MODULE_DEVICE_TABLE(of, spidev_dt_ids);
